src/app/modules/registration/components/send-button/send-button.component.ts

```diff
@@ -81,64 +88,64 @@ export class SendButtonComponent implements OnInit, OnChanges {
     }
   }
 
-  async requestDeleteDraft(): Promise<void> {
+  async requestDeleteDraft(): Promise<boolean> {
     const translations = await firstValueFrom(this.translateService
       .get([
-        'REGISTRATION.DELETE.DRAFT.HEADER',
         'REGISTRATION.DELETE.DRAFT.MESSAGE_NEW',
         'REGISTRATION.DELETE.DRAFT.MESSAGE_EDIT',
         'DIALOGS.YES',
         'DIALOGS.NO'
       ]));
-    const alert = await this.alertController.create({
-      header: translations['REGISTRATION.DELETE.DRAFT.HEADER'],
-      message:
-      this.draft.regId
-        ? translations['REGISTRATION.DELETE.DRAFT.MESSAGE_EDIT']
-        : translations['REGISTRATION.DELETE.DRAFT.MESSAGE_NEW'],
-      buttons: [
-        {
-          text: translations['DIALOGS.NO'],
-          role: 'cancel',
-          cssClass: 'secondary'
-        },
-        {
-          text: translations['DIALOGS.YES'],
-          handler: () => {
-            this.deleteDraft();
+
+    const message = this.draft.regId
+      ? 'REGISTRATION.DELETE.DRAFT.MESSAGE_EDIT'
+      : 'REGISTRATION.DELETE.DRAFT.MESSAGE_NEW';
+    return await this.confirmationModalService.askForConfirmation({
+      message: message, header: 'REGISTRATION.DELETE.DRAFT.HEADER', opts: {
+        buttons: [
+          {
+            text: translations['DIALOGS.NO'],
```

Comment:
Hadde vært digg om vi kunne sende inn nøkkel til språkfila i stedet for teksten på knapper, akkurat slik vi gjør for message og header. Da hadde vi sluppet å hente tekstene fra språkfila her og heller overlatt det til popup'en.